vars/nodePod.groovy

```diff
@@ -144,6 +144,77 @@ spec:
 """ ){ node(podLabel) { body() } }
       break
 
+    // CASE STANDARDBUILDER, MOUNT DOCKER SOCK
+    case "standardbuilder":
+      podTemplate(name: podLabel,
+        label: podLabel,
+        namespace: "jenkins",
+        yaml: """
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: jenkins-slave
+spec:
+  securityContext:
+    privileged: true
+    runAsUser: 0
+    fsGroup: 0
+  containers:
+    - name: jnlp
+      image: jenkins/jnlp-slave:alpine
+      securityContext:
+        privileged: true
+        runAsUser: 0
+        fsGroup: 0
+      tty: true
+      volumeMounts:
+        - name: dockersock
+          mountPath: "/var/run/docker.sock"
+        - name: workspace
+          mountPath: "/home/jenkins/workspace/${env.JOB_NAME}"
+        - name: tmp
+          mountPath: "/tmp"
+    - name: builder
+      image: accelbyte/golang-builder:1.12-alpine3.9
```

Comment:
i think we can just use `https://hub.docker.com/r/accelbyte/alpine` image for standard-builder, since we are not use golang for our job. WDYT?